### PR TITLE
Fix undefined variables

### PR DESCRIPTION
**Pull request by webmake**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Fix undefined variables
| Type?         | bug fix
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 
| How to test?  |

### CODE REVIEW

**Comment by webmake** [comment]
I've just found undefined variables, which can cause unexpected execution, so must be reviewed who understands better system.
Maybe those fixes are unnecessary and can be removed at all instead o fixing with proper named variables ?

File: src/Core/Addon/Theme/ThemeManager.php [hunk 1]
```diff
@@ -257,7 +257,7 @@ private function installFromZip($source)
         Tools::ZipExtract($source, $sandboxPath);
 
         $directories = $this->finder->directories()
-                                    ->in($sandbox_path)
+                                    ->in($sandboxPath)
```
Comment (mickaelandrieu) [inline src/Core/Addon/Theme/ThemeManager.php]: Already fixed,  can you rebase against develop branch?

**Comment by webmake** [comment]
Updated my develop branch with presta develop and merged into this feature/undefined_variables branch :)

**Comment by xBorderie** [comment]
You have keen eye, thank you this PR!
Only thing, part of your PR corrects the variable names, while another part of your PR change the way the tests work. Could you redo your work into two PRs? Thanks!

**Comment by maximebiloe** [comment]
Thank you @webmake

**Pull request merged**